packages/lit/src/lib/Theme/index.test.ts

```diff
@@ -41,4 +67,28 @@ describe('Theme', () => {
     const css = await theme.toCSS({ resolved: true });
     expect(css).toMatchSnapshot();
   });
+
+  it('can export to JSON token file', async () => {
+    const theme = new Theme();
+    const json = await theme.toTokensJSON();
+    return expect(json).toMatchSnapshot();
```

Comment:
```suggestion
    expect(json).toMatchSnapshot();
```

Fun thing, I had an eslint-vitest setting this morning that would warn against this. Maybe I should've enabled it 😆